controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -43,6 +45,9 @@ const (
 	StorageConsumerAnnotation     = "ocs.openshift.io.storageconsumer"
 	StorageClaimAnnotation        = "ocs.openshift.io.storageclaim"
 	StorageCephUserTypeAnnotation = "ocs.openshift.io.cephusertype"
+	MaxPGPerOSD                   = 600
+	MaxPGUnitSize                 = 128
+	StorageDeviceName             = "default"
```

Comment:
Same as the comment I added above for the other constant named `StorageDeviceName`